### ISSUE TITLE
Release Homebrew package

### ISSUE THREAD

**Issue by leontepe** [kamiyaa/joshuto#122]
Homebrew is another package manager for macOS. Arguably more popular than MacPorts. Would be nice to have the package released there too. Don't know how that works exactly, though.

**Comment by kamiyaa** [comment]
Yeah, that's a good idea. I'm probably not going to be maintaining it myself though (and I don't use Mac as my main system). Hopefully someone comes along and takes on the task of maintaining a package of joshuto for homebrew

**Comment by rasmus-kirk** [comment]
Not homebrew, but it is on nix which supports mac:
https://search.nixos.org/packages?channel=unstable&from=0&size=50&sort=relevance&type=packages&query=joshuto
